NuGet.Config

```diff
@@ -2,13 +2,13 @@
 <configuration>
     <packageSources>
         <clear />
-        <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
+        <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
```

Comment:
Had changed this briefly to see if it would unblock dependabot for this repo but unfortunately it did not and also doesn't have all the necessary packages.